src/flint/fmpz.jl

```diff
@@ -1139,19 +1139,22 @@ end
 issquare(x::fmpz) = Bool(ccall((:fmpz_is_square, :libflint), Cint,
                                (Ref{fmpz},), x))
 
+isprime(x::UInt) = Bool(ccall((:n_is_prime, :libflint), Cint, (UInt,), x))
+
 @doc Markdown.doc"""
-    is_prime(x::UInt)
+    isprime(x::fmpz)
 > Return `true` if $x$ is a prime number, otherwise return `false`.
 """
-is_prime(x::UInt) = Bool(ccall((:n_is_prime, :libflint), Cint, (UInt,), x))
+isprime(x::fmpz) = Bool(ccall((:fmpz_is_prime, :libflint), Cint,
+                              (Ref{fmpz},), x))
 
-# flint's fmpz_is_prime doesn't work yet
 @doc Markdown.doc"""
-    isprime(x::fmpz)
+    is_prime(x::Int)
```

Comment:
I think you meant `isprime` here, but this docstring can now be removed as redundant.